components/motor/gpio/encoded.go

```diff
@@ -173,14 +173,18 @@ func (m *EncodedMotor) makeAdjustments(ctx context.Context, goalRPM, goalPos, di
 		var currentRPM float64
 		if deltaTime == 0.0 {
 			currentRPM = 0
+			zeroRPMTracker++
+			m.logger.Debug("zero time delta calculated between motor power adjustments")
 		} else {
 			currentRPM = deltaPos / deltaTime
 			if currentRPM == 0 {
 				zeroRPMTracker++
 			}
 		}
-		if zeroRPMTracker > 10 {
-			m.logger.Errorf("error running motor %v, desired rpm is too low for stable motion: %v", m.Name().Name, goalRPM)
+		if zeroRPMTracker > 20 {
+			m.logger.Warnf("error running motor %v, current rpm is too low for stable motion: %v, asked to go at %v rpm",
```

Comment:
[nit - don't change this if you don't feel the need to] I don't know how I feel about this log change. I think it's kind of confusing because if I was a user I would think, why is the motor not going the speed I requested? but the point is that the speed requested causes unstable motion, which could cause the current speed to be something unexpected. I guess I don't know if we need to expose what the unexpected current speed is to the user. Also this will just be the most recent currentRPM after 20 iterations of instability, which is just a single point of the overall movement, which I would expect to always be 0 anyways?

Reply:
That's a good discussion point, if we're still going at near zero, and we're told the user their desired rpm is too low, which in the case naomi was seeing was calculated at 60 rpm from a base goign at 100 degspersecond for a 90 degree spin and she reported that it also happened when requesting more speed, then we've actually hit a motor's stall point or soemthing is topping it from moving and the calculated rpm is near zero because of a lack of position change. This could occur because of a bad jumper, or busted encoder line, or actually stalling against something. Then we are actually confusing the user more. 

I do want to test the drive in a square script if there are cases where this is happening, but the motor not moving along with an error makes it seems like something is returning in an erroneous state. I will consider other options once I come in and look at this with the rover performing.